tools/cli/pkg/ers/command/migrate_all.go

```diff
@@ -131,13 +132,17 @@ func (c *MigrationAllCommand) simpleWorker(workerId int, workChannel chan ers.Wo
 			workerId, work.Instance.Name, work.ProcessedCnt)
 		instance := work.Instance
 		refreshed, err := c.ersClient.GetOne(instance.Id)
-		if err != nil {
+		if err != nil || refreshed == nil {
 			c.log.Warnf("[Worker %d] GetOne error: %s", workerId, err.Error())
```

Comment:
err can be nil now, meaning `err.Error()` can panic, might be worth checking that